README.md

```diff
@@ -122,13 +145,20 @@ e.g.
     "title_number": "DN100",
     "borrowers": [
         {
-            "forename": "fred",
-            "middle_name": "joe",
-            "surname": "bloggs"
+            "forename": "lisa",
+            "middle_name": "ann",
+            "surname": "bloggette",
+            "address": "example address with postcode, PL14 3JR",
+            "dob": "23/01/1986",
+            "phone_number": "07502154062"
         },
         {
-            "forename": "lisa",
-            "surname": "bloggette"
+            "forename": "frank",
+            "middle_name": "ann",
+            "surname": "bloggette",
+            "address": "Example address with postcode, PL14 3JR",
+            "dob": "23/01/1986",
```

Comment:
Was that a user facing capture? Internal data formats don't force user facing format, we can easily convert between the two. In the meantime, to ensure portability of data, standards should be used where appropriate. This would be a totally appropriate place to use ISO_8601.